Application/global_settings_manager.py

```diff
@@ -58,8 +58,8 @@ def update_saved_settings(self):
             settings.endArray()
         settings.endArray()
 
-        settings.endGroup() # global-settings
         settings.endGroup() # encoding-buttons
+        settings.endGroup() # global-settings
```

Comment:
In lines 61-62, add an additional space between the closing parenthesis and the comment, according to PEP 8: E261 (2 spaces before inline comments)